test/runtests.bat

```diff
@@ -22,4 +25,10 @@ REM run the program, feed commands from input.txt file and redirect the output t
 java -classpath ..\bin seedu.addressbook.AddressBook < input.txt >> actual.txt
 
 REM compare the output to the expected output
-FC actual.txt expected.txt
\ No newline at end of file
+FC actual.txt expected.txt
+exit /b 1
+
+:errorHandling (
+echo ********** BUILD FAILURE ********** 
```

Comment:
Actually, maybe this logic can just be right after the `javac` call? Putting related things closer together to each other makes it easier to read the code. Separation can be useful if called multiple times, but this is not the case here.